Lib/test/test_codecs.py

```diff
@@ -1762,6 +1762,58 @@ def test_file_closes_if_lookup_error_raised(self):
 
             file().close.assert_called()
 
+    def test_copy(self):
+        orig = codecs.lookup('utf-8')
+        dup = copy.copy(orig)
+        self.assertIsNot(dup, orig)
```

Comment:
Add also the equality test. Here and in other test methods.
```suggestion
        self.assertIsNot(dup, orig)
        self.assertEqual(dup, orig)
```